internal/pkg/manifest/env.go

```diff
@@ -494,25 +498,20 @@ func (cfg *EnvironmentConfig) ELBAccessLogs() (*ELBAccessLogsArgs, bool) {
 	return &accessLogs.AdvancedConfig, true
 }
 
-// ALBSecurityGroupsConfig represents security group configuration settings for an ALB.
-type ALBSecurityGroupsConfig struct {
-	Ingress Ingress `yaml:"ingress"`
+// DeprecatedALBSecurityGroupsConfig represents security group configuration settings for an ALB.
```

Comment:
What do you think of moving these deprecated types under a new file: `deprecated.go`?

Reply:
good idea.